### DISCUSSION TITLE
Remember BRouter profile int/ext

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#270] [Q&A] [answered]
My problem:
With the motorcycle I use an external BRouter profile.
With the car I use the internal Cruiser-BRouter profile.
If I have selected the external profile for the motorcycle and switched to car, the external profile is still active.
You have to swap the BrouterProfile(int/ext) every time you switch from a car to a motorcycle.
That's laborious.
Could it be possible for Cruiser to remember the selected BRouter profile for the corresponding "vehicle"?

**Comment by devemux86** [comment, marked as answer]
@Leeb64 we already have a similar discussion, please see my answer there:

- https://github.com/devemux86/cruiser/discussions/266

The problem is the external files and the Android restrictions.
To reload an external file, user must grant permission via Android file browser.
